src/app/home/shared/forms/transaction-form/transaction-form.component.scss

```diff
@@ -1,12 +1,11 @@
 @import "../../../../../assets/styles/colors";
 
 :host {
+  top: 2rem;
   width: 100%;
-  height: 100%;
-  min-height: 100vh;
+  height: calc(100% - 2rem);
```

Comment:
🥇 This is wonderful! 💯 
...but...
You could make a variable for that top offset (2rem), and use is on the `top` and in that `height`.